pkg/integrations/cloudwatch_exporter/config.go

```diff
@@ -179,6 +179,11 @@ func PatchYACEDefaults(yc *yaceConf.ScrapeConf) {
 			metric.Delay = 0
 		}
 	}
+	for _, staticConf := range yc.Static {
+		for _, metric := range staticConf.Metrics {
+			metric.Delay = 0
+		}
+	}
```

Comment:
this is needed since upstream started applying defaults to static config as well since: https://github.com/nerdswords/yet-another-cloudwatch-exporter/pull/1039